status-whisper-usage-spec.md

```diff
@@ -59,6 +60,14 @@ Whisper's Proof Of Work algorithm is used to deter denial of service and various
 * proof-of-work requirement not larger than `0.002`
 * time-to-live not lower than `10` (in seconds)
 
+## Rate limiting
+
+In order to provide a very basic Denial-of-Service attack protection, there SHOULD be a rate limiting applied to leaf Whisper nodes, i.e. nodes not being controler within a cluster.
```

Comment:
what's the 'leaf Whisper node' distinction about? strikes me as odd, a protocol shouldn't take something like "a cluster" as for granted IMO

controller/controlled typo

Reply:
I guess leaf node is the wrong wording here? 
Are there actual leaf nodes right now? Doesn't each node currently have multiple peers (although perhaps always other cluster nodes?).

So it is rather "whitelisted cluster nodes" versus "the rest". I don't think I like that distinction.

Reply:
I guess leaf node is the wrong wording here?
Are there actual leaf nodes right now? Doesn't each node currently have multiple peers (although perhaps always other cluster nodes?).

So it is rather "whitelisted cluster nodes" versus "the rest". I don't think I like that distinction.

Reply:
Yeah, the current Status client nodes are are leaf nodes but this term is not used anywhere. The other is light nodes so the nodes which do not broadcast messages.

> So it is rather "whitelisted cluster nodes" versus "the rest". I don't think I like that distinction.

This is the most accurate. Not sure we can do it other way around. This is the reality where we have nodes which produce messages (and we want to rate limit them) and nodes which pass them along and we don't want limit these.

Reply:
I agree that this should be a generic setting on the node, which is tweaked based on the context.